src/test/functional/pages/BuzzPage.ts

```diff
@@ -24,7 +34,26 @@ export class BuzzPage extends BasePage {
         await this.submitPhoto.click();
     }
 
-   async deleteTheNewestPost(): Promise<void> {
+    async shareVideo(title:string, videoUrl:string): Promise<void> {
+      const generateIt = (request: any) => {
+        return request.url().startsWith('https://jnn-pa.googleapis.com') && request.url().endsWith('/GenerateIT');
+      };
+      await this.page.waitForLoadState('networkidle', { timeout: 7000 })
+      await this.shareVideoButton.click();
+      await this.shareVideoParagraph.type(title)
+      await this.pasteUrlTextarea.type(videoUrl)
+      await this.page.waitForRequest(generateIt)
+      await this.submitPhoto.click()
+      await this.page.waitForRequest(generateIt)
+  }
+
+   async deleteTheNewestPost(video: boolean | null): Promise<void> {
```

Comment:
the name for the boolean param should be maybe containsVideo? I think a better name for an boolean parameter

Reply:
agree, will change